jdaviz/configs/specviz/plugins/unit_conversion/unit_conversion.vue

```diff
@@ -60,27 +73,29 @@
       ></v-text-field>
     </v-row>
 
-    <v-row v-if="config == 'cubeviz'">
-      <v-divider></v-divider>
-    </v-row>
+    <div v-if="config == 'cubeviz'">
+      <v-row>
+        <v-divider></v-divider>
+      </v-row>
 
-    <v-row  v-if="config == 'cubeviz'">
-      <v-select
-        :menu-props="{ left: true }"
-        attach
-        :items="spectral_y_type_items.map(i => i.label)"
-        v-model="spectral_y_type_selected"
-        :label="api_hints_enabled ? 'plg.spectral_y_type =' : 'Spectral y-axis Type'"
-        :class="api_hints_enabled ? 'api-hint' : null"
-        hint="Select the y-axis physical type for the spectrum-viewer."
-        persistent-hint
-      ></v-select>
-    </v-row>
+      <v-row>
+        <v-select
+          :menu-props="{ left: true }"
+          attach
+          :items="spectral_y_type_items.map(i => i.label)"
+          v-model="spectral_y_type_selected"
+          :label="api_hints_enabled ? 'plg.spectral_y_type =' : 'Flux or Surface Brightness'"
```

Comment:
I think you missed this string in the rebase, unless you meant to change it back from the more general language.

Reply:
nope, missed that too 🤦   Reverted to "Spectral y-axis Type"